dapp/src/theme/utils/semanticTokens.ts

```diff
@@ -3,9 +3,10 @@ export const semanticTokens = {
     header_height: 24,
     modal_shift: "9.75rem", // 156px
     dashboard_card_padding: 5,
+    page_content_padding_x: 10,
```

Comment:
Non blocking
I have mixed feelings about these "tokens". We don't even reuse it elsewhere.
It only makes the need to jump between files as I'd like to investigate the value read from component JSX.